ragna/deploy/_ui/styles.py

```diff
@@ -11,109 +11,69 @@ def divider():
-                            height: var(--ragna-accordion-header-height);
-                            padding-top: 0px;
-                            padding-bottom: 0px;
-                            outline:0px;
-                            margin-left: 15px;
-                            margin-right: 15px;
-                            width: calc(100% - 30px);
-                            border-bottom: 2px solid #D9D9D9;
-                        
-                        }
-                        
-                        button.accordion-header div.card-button {
-                            font-size: 11px;
-                            padding-top: 5px;
-                            margin-left: 0px;
-                            margin-right: 10px;
-                        }
-                        
-                        div.card-header-row {
-                            height: var(--ragna-accordion-header-height);
-                            background-color: unset !important;
-                        }
-
-                        /* styles the content of the sources content (the expanding areas of the Accordion) */
-                        div.bk-panel-models-markup-HTML.markdown {
-                            margin-left: 15px;
-                            margin-right: 15px;
-                            margin-top:0px;
-                        }
-
-                    """
-        ]
-    }
-
-    pn.theme.fast.Fast.modifiers[pn.pane.HTML] = {
-        "stylesheets": [
-            """ :host(.card-title) {
-                                height: var(--ragna-accordion-header-height);
-                                margin: 0px;
-                            }
-                        
-                            :host(.card-title) div {
-                                height: var(--ragna-accordion-header-height);
-                                
-                                display:flex;
-                                flex-direction:row;
-                                align-items:center;
-                            }
-                        
-
-                            :host(.card-title) h3 {
-                                font-weight: normal;
-                            }
-                        """
-        ]
-    }
-
-    pn.theme.fast.Fast.modifiers[pn.pane.Markdown] = {
-        "stylesheets": [
-            """  /* Styles the content of the sources content (the expanding areas of the Accordion).
-                            This fixes a small margin-top that is added by default and that leads to overflowing content 
-                            in some cases.
-                            */
-                            :host(.source-content) p:nth-of-type(1)  {
-                                margin-top: 0px;
-                            }
-                        """
-        ]
-    }
+    add_modifier(
+        pn.layout.Accordion, " :host(.source-accordion) { height: 100%; width: 100%; } "
```

Comment:
Should this also go into a CSS file? If not, let's use our helper function.

Reply:
For the sake of keeping as much CSS as possible out of the python code, this should move into CSS files. I was hesitating at first because that's a very short snippet, but it's actually the right thing to do considering the goal. 
I'll also move the raw CSS from `apply_design_modifiers_global` into CSS files, that's the same pattern.